resources/views/MyAccount/Components/OrderHistoryItem.twig

```diff
@@ -0,0 +1,59 @@
+{{ component( "Ceres::MyAccount.Components.ChangePaymentMethod" ) }}
+
+<script type="x/template" id="vue-order-history-item">
+    <article class="cmp cmp-order-item">
+        <div class="item-details small">
+            <div class="mb-3">
+                <h5>{{ trans("Ceres::Template.orderHistoryOrderId") }} ${ order.order.id }</h5>
+            </div>
+
+            <div class="row">
+                <div class="col-12 col-md-8">
+                    <div class="row">
+                        <strong class="col-6 col-sm-5">{{ trans("Ceres::Template.orderHistoryDate") }}</strong>
+                        <span class="col-6 col-sm-7">{{ entry.order.createdAt | formatDateTime | date( trans("Ceres::Template.devDateTimeFormat") ) }} {{ trans("Ceres::Template.orderHistoryTime") }}</span>
+                    </div>
+                    <div class="row my-account-no-line">
+                        <strong class="col-6 col-sm-5">{{ trans("Ceres::Template.orderHistoryStatus") }}</strong>
+                        <span class="col-6 col-sm-7">${ order.order.statusName }</span>
+                    </div>
+                        <div class="row" v-if="order.paymentMethodName.trim().length > 0">
+                            <strong class="col-6 col-sm-5">{{ trans("Ceres::Template.orderHistoryPaymentMethod") }}</strong>
+                            <span class="col-6 col-sm-7" :id="'payment_name_' + order.order.id">${ order.paymentMethodName }</span>
+                        </div>
+                    <div class="row">
+                        <strong class="col-6 col-sm-5">{{ trans("Ceres::Template.orderHistoryPaymentStatus") }}</strong>
+                        <span class="col-6 col-sm-7" :id="'payment_state_' + order.order.id">
+                            ${ 'Ceres::Template.orderHistoryPaymentStatus_' + order.paymentStatus | translate() }
+                        </span>
+                    </div>
+                </div>
+
+                <div class="col-12 col-md-4">
+                    <div class="payment-change-box">
```

Comment:
Die Benennung der Klasse ist missverständlich. Der Container enthält ja mehr, als nur den Button zum Ändern der Zahlungsart. Bitte allgemeingültigere Bezeichnung wählen

Reply:
Die Datei ist nicht in benutzung und wird gelöscht.